src/util/orders.ts

```diff
@@ -0,0 +1,37 @@
+import { assetDataUtils, BigNumber, generatePseudoRandomSalt, Order } from '0x.js';
+
+import { FEE_RECIPIENT, MAKER_FEE, TAKER_FEE, ZERO_ADDRESS } from '../common/constants';
+import { OrderSide } from '../util/types';
+
+interface BuildOrderParams {
+    account: string;
+    tokenAddress: string;
+    wethAddress: string;
+    amount: BigNumber;
+    price: number;
+    exchangeAddress: string;
+}
+
+export const buildOrder = (params: BuildOrderParams, side: OrderSide): Order => {
+    const { account, exchangeAddress, amount, price, tokenAddress, wethAddress } = params;
+    const tomorrow = new BigNumber(Math.floor(new Date().valueOf() / 1000) + 3600 * 24);
```

Comment:
We can create a ticket later to adopt a constant value or something set in the environment (NNNN mins). Not really urgent right now.

Reply:
Oh, yes, I totally forgot about this :man_facepalming: An environment variable seems the way to go.

Reply:
:+1: this should be a non-blocking detail. If you want, just create an issue to deal with it later.